README.md

```diff
@@ -36,6 +36,7 @@ The following types of sequence are available with the following parameters bein
 - `Prime()`
 - `Triangle()`
 - `Twin(a, b)`
+- `NthPower(n)`
```

Comment:
You need to update the `index.ts` file in the `src` folder if you want this to be available to consumers.